apps/dashboard/src/components/contract-components/contract-publish-form/default-factory.tsx

```diff
@@ -66,9 +77,36 @@ export const DefaultFactory: React.FC<DefaultFactoryProps> = ({
             }
           />
         </FormControl>
+        {shouldShowExtensionsParamInput && (
+          <>
+            <Flex flexDir="column" gap={2}>
+              <Heading size="title.md">Extension Param</Heading>
```

Comment:
"Param" is not something we should ever write into public facing copy

Reply:
++ @Yash094

Reply:
++ @kumaryash90